web/src/styles/onboarding.css

```
@@ -2711,3 +3282,485 @@
+
+  @starting-style {
+    opacity: 0;
+    transform: translateY(14px) scale(0.985);
+    filter: blur(4px);
+  }
+}
+/* Browsers without `interpolate-size` ignore the height transition but
+   still honour opacity/translate — fall back to a plain fade-up via
+   keyframe so those users still get a smooth entrance. */
+@supports not (interpolate-size: allow-keywords) {
+  @keyframes onboardingMessageEnter {
```

Stylelint (17.12.0)

[error] Expected keyframe name "onboardingMessageEnter" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)